src/examples/libpmemobj/pmemlog/obj_pmemlog_simple.c

```diff
@@ -47,21 +47,20 @@
  *   t - tell
  * "a", "w" and "v" require a parameter string(s) separated by a colon
  */
-
+#include <ex_common.h>
 #include <sys/stat.h>
 #include <string.h>
 #include <stdio.h>
 #include <assert.h>
 #include <stdlib.h>
 #include <errno.h>
-#include <sys/param.h>
-
+#define MIN(a, b)  (((a) < (b)) ? (a) : (b))
```

Comment:
I just searched our tree for "#define MIN". I found 5, adding sixth is really not needed.